nslsii/detectors/xspress3v33.py

```diff
@@ -0,0 +1,647 @@
+from __future__ import print_function, division
```

Comment:
Doesn't the old version also use EPICS areaDetector? I believe this new version is meant to take advantage of the improved APS version of the xspress3 driver on github: https://github.com/epics-modules/xspress3, which relies on areaDetector version R3-3 or higher, instead of the vendor driver: https://github.com/quantumdetectors/xspress3-epics, which uses areaDetector version R1-9-1. I think this is where the v33 comes from

Reply:
Thanks for the details, @jwlodek. My understanding of the old IOCs was not correct, and indeed, it seems to use the [AD R1-9-1](https://github.com/quantumdetectors/xspress3-epics/blob/3d697abf4f1eadd708c677e28c6378f4c3972a41/configure/RELEASE#L27) (given it is the deployed version at NSLS-II):
```
AREA_DETECTOR=  $(SUPPORT)/areaDetectorR1-9-1
```

Reply:
The "old" xspress3 code in this package does not use the ophyd areadetector classes, even though the corresponding IOC uses areaDetector 1-9-1. Did I say that right?

Reply:
Now that I've had some time to think about it I would like to address the naming issue by replaceing `nslsii/detectors/xspress3v33.py` with `nslsii/areadetector/xspress3.py`. This sort-of mirrors `ophyd/areadetector/detectors.py`. I prefer `xspress3.py` in this case because there will be a lot of xspress3 code here and I would like to keep it separate from code for other detectors that might land in `nslsii/areadetector`.

Reply:
@jklynch I believe the old xspress3 code also uses ophyd areaDetector classes, see the [imports in the module](https://github.com/NSLS-II/nslsii/blob/0e4e463d95d1c634f16980323c0de45312f881b0/nslsii/detectors/xspress3.py#L12). The difference is that it uses older versions of the central ophyd classes that reflect the older PV naming structure. Also any features exclusive to the APS driver are not included, since the current `xspress3.py` reflects the quantumdetectors variant.

Perhaps it would be best to differentiate the two based on the driver they support? Ala `xspress3APS.py` and `xspress3QD.py`?

Reply:
I prefer not to change `nslsii/detectors/xspress3.py` because it has been imported in many places outside the `nslsii` package that would have to be changed.

My hope is that the Quantum IOC will be replaced at most beamlines in the not-too-distant future, which would (eventually) make `xspress3APS.py` redundant, which is why I prefer to leave "APS" off.